app/src/main/java/com/tab/mybadges/MockBadgeListViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.tab.mybadges
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.tab.mybadges.entities.Badge
+
+object MockBadgeListViewModel: ViewModel() {
```

Comment:
To return a mock badge list, I think you should be already creating the view model correctly. For example, you could have had a `BadgeListViewModel: ViewModel()` and have a function which could be `getBadges()'. That way, in your view, you would already have the code you need and would not need to change it once you are ready to get the badges without mocking it.

Reply:
@viniciusalbuquerque I've updated the code regarding your comment, can you review it again?